litestar/constants.py

```diff
@@ -14,6 +17,7 @@
 HTTP_RESPONSE_BODY: Final = "http.response.body"
 HTTP_RESPONSE_START: Final = "http.response.start"
 ONE_MEGABYTE: Final = 1024 * 1024
+OPENAPI_JSON_HANDLER_NAME: Final = f"{uuid4().hex}_litestar_openapi_json"
```

Comment:
The UUID is to prevent users being able to depend on the absolute value and also prevent collision to user-space names, but I'm not married to the idea either if anyone thinks it OTT.